build-kmod-nvidia.sh

```diff
@@ -20,20 +21,25 @@ rpm -qa |grep nvidia
 KERNEL_VERSION="$(rpm -q "${KERNEL_NAME}" --queryformat '%{VERSION}-%{RELEASE}.%{ARCH}')"
 NVIDIA_AKMOD_VERSION="$(basename "$(rpm -q "akmod-nvidia" --queryformat '%{VERSION}-%{RELEASE}')" ".fc${RELEASE%%.*}")"
 
+if [[ "${KERNEL_MODULE_TYPE}" == "open" ]]; then
+    sed -i -e 's/kernel$/kernel-open/g' /etc/nvidia/kernel.conf
+fi
 
 akmods --force --kernels "${KERNEL_VERSION}" --kmod "nvidia"
 
 modinfo /usr/lib/modules/${KERNEL_VERSION}/extra/nvidia/nvidia{,-drm,-modeset,-peermem,-uvm}.ko.xz > /dev/null || \
 (cat /var/cache/akmods/nvidia/${NVIDIA_AKMOD_VERSION}-for-${KERNEL_VERSION}.failed.log && exit 1)
 
+# View license information
+modinfo -l /usr/lib/modules/${KERNEL_VERSION}/extra/nvidia/nvidia{,-drm,-modeset,-peermem,-uvm}.ko.xz
```

Comment:
While this lists it out, can we do a condition check to make sure that the correct licensed kmod was built given the input.

Reply:
Yes please

Reply:
It looks like we requested to do a check here but we didn't?